### PR TITLE
[zh] Fix link to dev@kubernetes mailing list

### PR DESCRIPTION
**Pull request by jberkus**
Update the link to the new Dev Googlegroup. Submitting one PR per localization per @sftim

### CODE REVIEW

File: content/zh/community/_index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <!-- <div class="content">
             <h3>Ensuring Kubernetes works well everywhere and for everyone.</h3>
             <p>Connect with the Kubernetes community on our <a href="http://slack.k8s.io/">Slack channel</a>, <a href="https://discuss.kubernetes.io/">discussion board</a>, or join the
-                <a href="https://groups.google.com/forum/#!forum/kubernetes-dev">Kubernetes-dev Google group</a>. A weekly
+                <a href="https://groups.google.com/a/kubernetes.io/g/dev/">eev@kubernetes Google group</a>. A weekly
```
Comment (jihoon-seo) [inline content/zh/community/_index.html:20]:
```suggestion
                <a href="https://groups.google.com/a/kubernetes.io/g/dev/">dev@kubernetes Google group</a>. A weekly
```

**Comment by jihoon-seo** [comment]
/retitle [zh] Fix link to dev@kubernetes mailing list

File: content/zh/community/_index.html [hunk 2]
```diff
@@ -30,7 +30,7 @@ <h3>Ensuring Kubernetes works well everywhere and for everyone.</h3>
               <h3>保证 Kubernetes 到处都适用，每个人都喜欢。</h3>
               <p>在我们的<a href="http://slack.k8s.io/">Slack channel</a>,
                 <a href="https://discuss.kubernetes.io/">讨论版</a>, 或者
-                <a href="https://groups.google.com/forum/#!forum/kubernetes-dev">Kubernetes-dev Google 群主</a>上和 Kubernetes 互动。
+                <a href="https://groups.google.com/a/kubernetes.io/g/dev/">dev@kubernetes Google 群主</a>上和 Kubernetes 互动。
```
Comment (idealhack) [inline content/zh/community/_index.html:33]:
```suggestion
                <a href="https://groups.google.com/a/kubernetes.io/g/dev/">dev@kubernetes Google 群组</a>上和 Kubernetes 互动。
```

**Comment by tengqm** [comment]
Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this **on the main branch**:

```shell
./scripts/lsync.sh content/zh/community/_index.html
```

/hold

**Comment by jberkus** [comment]
@tengqm I'm sorry, I have no idea what you're asking for.  You seem to be asking me to merge all pending changes for the website?  Or something?

**Comment by tengqm** [comment]
@jberkus Thanks for proposing this fix. What I wanted to clarify is that the Chinese localization team as a protocol for synchronizing the English site which is always changing. The way we achieve this is as this. For every page that was already translated (say on Jan 1st), we need to ensure all changes to the English page since then to be included in the next translation. We don't accept partial syncs.

I think we can safely close this PR. The next time when someone is picking this page for a resync, this link will be fixed.

**Comment by idealhack** [comment]
@tengqm I get your point. But as for sig-contribex concerned, this is a communication channel change, the sooner we get this change in, the better.

Also, I see changes for other localizations are processing https://github.com/kubernetes/website/pulls?q=dev%40kubernetes+mailing+list

Can we merge this change first, then put the syncing change in a future PR?

**Comment by tengqm** [comment]
/close
Will fix this in a separate PR and make sure the page is fully sync'ed.

**Pull request closed without merging**